scripts/lib/desktop-rtl-herdr.ps1

```
@@ -0,0 +1,598 @@
+    param([Parameter(Mandatory)][string]$Exe)
+    try {
+        $psi = New-Object System.Diagnostics.ProcessStartInfo
+        $psi.FileName = $Exe
+        $psi.Arguments = '--version'
+        $psi.UseShellExecute = $false
+        $psi.RedirectStandardOutput = $true
+        $psi.RedirectStandardError = $true
+        $psi.CreateNoWindow = $true
+        $proc = [System.Diagnostics.Process]::Start($psi)
+        $out = $proc.StandardOutput.ReadToEnd() + $proc.StandardError.ReadToEnd()
+        if (-not $proc.WaitForExit(20000)) { try { $proc.Kill() } catch {} ; return $null }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)